<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WTC-5159473-l2" ignored="0" name="Test_Test" tests="16" failures="15" timestamp="11 Jul 2019 17:24:51 GMT" time="2136.711" errors="0">
  <testcase name="SetEvironmentLevel" time="0.055" classname="SupportClasses.Environment">
    <skipped/>
  </testcase> <!-- SetEvironmentLevel -->
  <testcase name="Create_Shipment" time="0.544" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="0.741" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="0.966" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="1.196" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="1.415" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="1.598" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="1.768" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="1.97" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="2.156" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="2.36" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="1.981" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="2.046" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="2.046" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="2.024" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="1.987" classname="SHPC_Application.SHPC_API_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{
    &amp;quot;error&amp;quot;: &amp;quot;invalid_request&amp;quot;,
    &amp;quot;error_description&amp;quot;: &amp;quot;validation error&amp;quot;
}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{
    "error": "invalid_request",
    "error_description": "validation error"
}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_API_Endpoints.Create_Shipment(SHPC_API_Endpoints.java:108)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
</testsuite> <!-- Test_Test -->
